### ISSUE TITLE
pre-built static binary install script does not detect SLES as systemd OS

### ISSUE THREAD

**Issue by tsingletonacic** [netdata/netdata#4641]
##### Bug report summary

Using the pre-built static binary on SLES (Suse Linux Enterprise Server) 12.3, I get an error at init file registration

 --- Install netdata at system init --- 
I don't know what init file to install on system 'sles-12.3'. Open a github issue to help us fix it.
 FAILED  Cannot install netdata init service. 

SLES 12.3 uses /usr/lib/systemd as systemd path instead of /lib/systemd

I have prepared a patch for functions.sh, will submit as soon as I understand how to

##### OS / Environment
SLES 12.3

```
# rpm -qa | grep systemd | sort 
grub2-systemd-sleep-plugin-2.02-2.12.noarch
libsystemd0-228-142.1.x86_64
libsystemd0-32bit-228-142.1.x86_64
systemd-228-142.1.x86_64
systemd-32bit-228-142.1.x86_64
systemd-bash-completion-228-142.1.noarch
systemd-presets-branding-SLE-12.2-2.26.noarch
systemd-sysvinit-228-142.1.x86_64
util-linux-systemd-2.29.2-2.2.x86_64
```

**Comment by tsingletonacic** [comment]
```
--- functions.sh.original	2018-11-13 15:08:01.589754691 +0100
+++ functions.sh	2018-11-13 15:02:34.901759130 +0100
@@ -441,8 +441,8 @@
 issystemd() {
     local pids p myns ns systemctl
 
-    # if the directory /lib/systemd/system does not exit, it is not systemd
-    [ ! -d /lib/systemd/system ] && return 1
+    # if the directory /lib/systemd/system OR /usr/lib/systemd/system (SLES 12.x) does not exit, it is not systemd
+    [ ! -d /lib/systemd/system -a ! -d /usr/lib/systemd/system ] && return 1
 
     # if there is no systemctl command, it is not systemd
     # shellcheck disable=SC2230
@@ -563,15 +563,25 @@
             NETDATA_START_CMD="systemctl start netdata"
             NETDATA_STOP_CMD="systemctl stop netdata"
 
+            SYSTEMD_DIRECTORY=""
+
             if [ -d "/lib/systemd/system" ]
             then
+                SYSTEMD_DIRECTORY="/lib/systemd/system"
+            elif [ -d "/usr/lib/systemd/system" ]
+            then
+                SYSTEMD_DIRECTORY="/usr/lib/systemd/system"
+            fi
+
+            if [ "${SYSTEMD_DIRECTORY}x" != "x" ]
+            then
                 echo >&2 "Installing systemd service..."
-                run cp system/netdata.service /lib/systemd/system/netdata.service && \
+                run cp system/netdata.service "${SYSTEMD_DIRECTORY}/netdata.service" && \
                     run systemctl daemon-reload && \
                     run systemctl enable netdata && \
                     return 0
             else
-                echo >&2 "no '/lib/systemd/system' directory; cannot install netdata.service"
+                echo >&2 "no systemd directory; cannot install netdata.service"
             fi
         else
             install_non_systemd_init
```

**Comment by Ferroin** [comment]
Confirmed also on OpenSUSE Leap and Tumbleweed.

**Comment by paulfantom** [comment]
Closing since #4643 is merged. Please reopen if it still doesn't work.

**Comment by paulfantom** [comment]
This should be fixed in #4643. Please reopen if it still doesn't work for you.
